src/components/Todos/DoinItGrid.spec.js

```diff
@@ -0,0 +1,44 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import DoinItGrid from './DoinItGrid';
+
+describe('Display a title-list of the array todos an a Headline', () => {
```

Comment:
WIthin the `describe` statement, you name the component you are testing. The specific test description is part of the `it` statement.
```suggestion
describe('DoinItGrid', () => {
```